tests/functional/spec/list_collector_primary_person.spec.js

```diff
@@ -95,7 +95,7 @@ describe("Primary Person List Collector Survey", () => {
 
     it("When the user submits, then they are allowed to submit the survey", () => {
       $(SubmitPage.submit()).click();
-      expect($(ThankYouPage.heading()).getText()).to.contain("Submission successful");
+      expect($(ThankYouPage.heading()).getText()).to.contain("Thank you for completing the Test PrimaryPersonListCollector");
```

Comment:
We should be checking we're on at the right url, rather than checking for text content here.

Reply:
This test is checking the url now, thanks!